inc/dynamic-css/related-posts.php

```diff
@@ -0,0 +1,191 @@
+				'color' => astra_get_foreground_color( $link_color ),
+			),
+			'.ast-related-posts-title-section .ast-related-posts-title' => array(
+				'color' => esc_attr( $related_posts_title_color ),
+			),
+			'.ast-single-related-posts-container, .ast-related-posts-wrapper .ast-related-posts-inner-section' => array(
+				'background-color' => esc_attr( $related_posts_bg_color ),
+			),
+			/**
+			 * Related Posts - Section Title
+			 */
+			'.ast-related-posts-title-section .ast-related-posts-title' => array(
+				'font-family'    => astra_get_css_value( $related_posts_section_title_font_family, 'font' ),
+				'font-weight'    => astra_get_css_value( $related_posts_section_title_font_weight, 'font' ),
+				'font-size'      => astra_responsive_font( $related_posts_section_title_font_size, 'desktop' ),
+				'line-height'    => esc_attr( $related_posts_section_title_line_height ),
+				'text-transform' => esc_attr( $related_posts_section_title_text_transform ),
+			),
+			/**
+			 * Related Posts - Post Title
+			 */
+			'.ast-single-post .ast-related-post-content .ast-related-post-title *' => array(
+				'font-family'    => astra_get_css_value( $related_post_title_font_family, 'font' ),
+				'font-weight'    => astra_get_css_value( $related_post_title_font_weight, 'font' ),
+				'font-size'      => astra_responsive_font( $related_post_title_font_size, 'desktop' ),
+				'line-height'    => esc_attr( $related_post_title_line_height ),
+				'text-transform' => esc_attr( $related_post_title_text_transform ),
+				'color'          => esc_attr( $related_post_text_color ),
+			),
+			/**
+			 * Related Posts - Meta
+			 */
+			'.ast-single-post .ast-related-post-content .entry-meta, .ast-single-post .ast-related-post-content .entry-meta *' => array(
+				'font-family'    => astra_get_css_value( $related_post_meta_font_family, 'font' ),
+				'font-weight'    => astra_get_css_value( $related_post_meta_font_weight, 'font' ),
+				'font-size'      => astra_responsive_font( $related_post_meta_font_size, 'desktop' ),
+				'line-height'    => esc_attr( $related_post_meta_line_height ),
+				'text-transform' => esc_attr( $related_post_meta_text_transform ),
+				'color'          => esc_attr( $related_posts_meta_color ),
+			),
+			'.ast-single-post .ast-related-post-content .entry-meta a:hover, .ast-single-post .ast-related-post-content .entry-meta span a span:hover' => array(
+				'color' => esc_attr( $related_posts_meta_link_hover_color ),
+			),
+			/**
+			 * Related Posts - CTA
+			 */
+			'.ast-single-post .ast-related-post-content .ast-related-post-cta a' => array(
+				'color' => esc_attr( $related_posts_link_color ),
+			),
+			'.ast-single-post .ast-related-post-content .ast-related-post-cta a:hover' => array(
+				'color' => esc_attr( $related_posts_link_hover_color ),
+			),
+			/**
+			 * Related Posts - Content
+			 */
+			'.ast-single-post .ast-related-post-content .ast-related-post-excerpt' => array(
```

Comment:
```suggestion
			'.ast-related-post-excerpt' => array(
```
Please check for all the CSS if we can remove the extra CSS classes if not required?

Reply:
Managed lower weightage CSS selectors [here](https://github.com/brainstormforce/astra/pull/2559/commits/5f9d157e536656b10c2327d1ece91b7b97dc6830)